sdk/monitor/Azure.Monitor.Query/src/MetricsClient.cs

```diff
@@ -39,11 +39,14 @@ public MetricsClient(Uri endpoint, TokenCredential credential, MetricsClientOpti
 
             _clientDiagnostics = new ClientDiagnostics(options);
 
-            var scope = "https://metrics.monitor.azure.com/.default";
+            var authorizationScope = $"{(string.IsNullOrEmpty(options.Audience?.ToString()) ? MetricsClientAudience.AzurePublicCloud : options.Audience)}";
+            authorizationScope += "//.default";
```

Comment:
I have 2 / in MetricsQueryClient so I was following the same pattern here. Should I add one / to the Metrics Audience strings?

Reply:
I think in the past, with the way audiences and scopes were handled before, the double slash was needed in some scenarios (generally with the ARM endpoint). However, the format is `{audience}/.default` ([ref](https://learn.microsoft.com/en-us/entra/identity-platform/scopes-oidc#the-default-scope)). In practice, I don't think it really matters anymore given the fact that it will still work with either one slash or two.

But for this PR, since you were using one slash before (i.e. `"https://metrics.monitor.azure.com/.default"`), let's maintain the usage of one slash.